ChangeLog

```diff
@@ -1,4 +1,66 @@
 
+Thu Mar 29 2018 Krzysztof Czurylo <krzysztof.czurylo@intel.com>
+	* Version 1.4
+	This is the first release of PMDK under a new name.
+	The NVML project has been renamed to PMDK (Persistent Memory
+	Development Kit).  This is only the project/repo name change
+	and it does not affect the names of the PMDK packages.
+	See this blog article for more details on the reasons and
+	impact of the name change:
+	http://pmem.io/2017/12/11/NVML-is-now-PMDK.html
+
+	New features:
+	- common: support for concatenated Device-DAX devices
+	  with 2M/1G alignment
+	- common: add support for MAP_SYNC flag
+	- common: always enable Valgrind instrumentation (#292)
+	- common: pool set options / headerless pools
+	- pmem: add support for "deep flush" operation
+	- rpmem: add rpmem_deep_persist
+	- doc: split man pages and add per-function aliases (#385)
+
+	Optimizations:
+	- pmem: skip CPU cache flushing when eADR is available
+	  (no Windows support yet)
+	- pmem: make use of AVX512F instructions in memcpy/memset (#656)
```

Comment:
It's not enabled by default, so it's a bit misleading to say "make use of AVX512F...".